home/models.py

```diff
@@ -605,6 +605,8 @@ def latest_revision_rating(self):
 
     @property
     def whatsapp_template_prefix(self):
+        if self.whatsapp_title:
+            self.whatsapp_title = self.whatsapp_title.lower()
```

Comment:
Do we want to set `self.whatsapp_title` to lowercase here, or just return a lowercase version of it?

Also, on the line below we call `.replace()` on it unconditionally. That means we don't need to check if it's non-empty before lowercasing it.